addons/calendar/models/calendar.py

```diff
@@ -445,7 +446,7 @@ def do_mail_reminder(self, alert):
 
         result = False
         if alarm.type == 'email':
-            result = meeting.attendee_ids.filtered(lambda r: r.state != 'declined')._send_mail_to_attendees('calendar.calendar_template_meeting_reminder', force_send=True)
+            result = meeting.attendee_ids.filtered(lambda r: r.state != 'declined')._send_mail_to_attendees('calendar.calendar_template_meeting_reminder', True, meeting)
```

Comment:
Do not use positional arguments. Instead:
```suggestion
            result = meeting.attendee_ids.filtered(lambda r: r.state != 'declined')._send_mail_to_attendees('calendar.calendar_template_meeting_reminder', recurring_event_id=meeting)
```